data/net/minecraft/world/inventory/ResultSlot.mapping

```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/world/inventory/ResultSlot
+	METHOD <init> (Lnet/minecraft/world/entity/player/Player;Lnet/minecraft/world/inventory/CraftingContainer;Lnet/minecraft/world/Container;III)V
+		ARG 1 player
+		ARG 2 craftingContainer
```

Comment:
Point of 'parameters follow field names': `craftingContainer` -> `craftingSlots`.